src/components/task-edit.js

```diff
@@ -0,0 +1,134 @@
+import {COLORS, DAYS, MONTH_NAMES} from '../const.js';
+import {formatTime} from '../utils.js';
+
+const createColorsMarkup = (colors, currentColor) => {
+  return colors
+    .map((color, index) => {
+      return (
+        `<input
+          type="radio"
+          id="color-${color}-${index}"
+          class="card__color-input card__color-input--${color} visually-hidden"
+          name="color"
+          value="${color}"
+          ${currentColor === color ? `checked` : ``}
+        />
+        <label
+          for="color-${color}--${index}"
+          class="card__color card__color--${color}"
+          >${color}</label
+        >`
+      );
+    })
+    .join(`\n`);
+};
+
+const createRepeatingDaysMarkup = (days, repeatingDays) => {
+  return days
+    .map((day, index) => {
+      const isChecked = repeatingDays[day];
+      return (
+        `<input
+          class="visually-hidden card__repeat-day-input"
+          type="checkbox"
+          id="repeat-${day}-${index}"
+          name="repeat"
+          value="${day}"
+          ${isChecked ? `checked` : ``}
+        />
+        <label class="card__repeat-day" for="repeat-${day}-${index}"
+          >${day}</label
+        >`
```

Comment:
Порнуха какая то переносить `>`
```suggestion
        <label class="card__repeat-day" for="repeat-${day}-${index}">${day}</label>`
```